### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by lincoln1883**
For this milestone, I was able to complete the following task:
- Generate and clone the repo.
- Professionally document the project.
- Write the SQL to create the database table schema.
- Write insertion instructions to populate the table.
- Correctly query the table.
 - Find all animals whose name ends in "mon". 
Screenshot from 2023-07-10 13-46-56
- List the name of all animals born between 2016 and 2019.
Screenshot from 2023-07-10 13-46-37
- List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot from 2023-07-10 13-53-04
- List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot from 2023-07-10 13-52-35
- List names and escape attempts of animals that weigh more than 10.5kg.
Screenshot from 2023-07-10 13-49-49
- Find all animals that are neutered.
Screenshot from 2023-07-10 13-50-42
- Find all animals not named Gabumon.
Screenshot from 2023-07-10 13-48-10
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot from 2023-07-10 13-47-35

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
+SELECT name from animals WHERE date_of_birth BETWEEN '2017-05-12' AND '2020-02-02';
```
Comment (Zuhaib042) [inline queries.sql]: - [ ] Kindly consider updating this query as the dates in your query are wrong. it should be `'2016/01/01' AND '2019/31/12'`  as in the following query :
    -  List the name of all animals born between 2016 and 2019.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @lincoln1883 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ well written queries
✅ Schema is on point

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (lincoln1883) [reply queries.sql]: - [x] Kindly consider updating this query as the dates in your query are wrong. it should be `'2016/01/01' AND '2019/31/12'`  as in the following query :
    -  List the name of all animals born between 2016 and 2019.
    - Edited the query to match the date range as specified.
    
Screenshot from 2023-07-10 14-28-44

**Review by Ibizugbe — APPROVED** [review]
Hi @lincoln1883 👋👋👋, 

VeryNiceNiceGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Queries have been updated as requested by the previous reviewer👍.
- [x] The image showing the query result has also been added to the thread👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**